muskathlon/controllers/main.py

```diff
@@ -50,90 +50,9 @@ def my_muskathlon_details(self, registration, **kwargs):
-        if form_id is None or form_id == about_me_form.form_id:
-            about_me_form.form_process()
-            form_success = about_me_form.form_success
-
-        kw["form_model_key"] = "cms.form.muskathlon.large.picture"
-        large_picture_form = self.get_form(
-            "advocate.details", advocate_details_id, **kw
-        )
-        if form_id is None or form_id == large_picture_form.form_id:
-            large_picture_form.form_process()
-            form_success = large_picture_form.form_success
-
-        kw["form_model_key"] = "cms.form.muskathlon.passport"
-        passport_form = self.get_form("event.registration", registration.id, **kw)
-        if form_id is None or form_id == passport_form.form_id:
-            passport_form.form_process()
-            form_success = passport_form.form_success
-
-        kw["form_model_key"] = "cms.form.muskathlon.flight.details"
-        kw["registration_id"] = registration.id
-        flight_type = kw.get("flight_type")
-        kw["flight_type"] = "outbound"
-        flight = registration.flight_ids.filtered(lambda f: f.flight_type == "outbound")
-        outbound_flight_form = self.get_form("event.flight", flight.id, **kw)
-        if form_id is None or (
-                form_id == outbound_flight_form.form_id
-                and (not flight_type or flight_type == "outbound")
-        ):
-            outbound_flight_form.form_process(**kw)
-            form_success = outbound_flight_form.form_success
-        kw["flight_type"] = "return"
-        flight = registration.flight_ids.filtered(lambda f: f.flight_type == "return")
-        return_flight_form = self.get_form("event.flight", flight.id, **kw)
-        if form_id is None or (
-                form_id == return_flight_form.form_id
-                and (not flight_type or flight_type == "return")
-        ):
-            return_flight_form.form_process(**kw)
-            form_success = return_flight_form.form_success
-
-        values.update(
-            {
-                "trip_info_form": trip_info_form,
-                "coordinates_form": coordinates_form,
-                "about_me_form": about_me_form,
-                "large_picture_form": large_picture_form,
-                "passport_form": passport_form,
-                "outbound_flight_form": outbound_flight_form,
-                "return_flight_form": return_flight_form,
-            }
-        )
-        values.update(kw)
-        if "registrations" not in values.keys():
-            registrations_array = []
-            for reg in partner.registration_ids:
-                registrations_array.append(reg)
-            values['registrations'] = registrations_array
-        # This fixes an issue that forms fail after first submission
-        if form_success:
-            result = request.redirect("/my/home")
-        else:
-            result = request.render("muskathlon.custom_portal_my_home", values)
```

Comment:
Is all this code removed transferred somwhere? Otherwise I fear that the Muskathlon functionalities won't work anymore?

Reply:
No I did not transfer the code anywhere. I did not look in details but it looked to me that this part of the code simply updated the values that were used to render the old _My Account_ page. I am not sure where we should put it if we want to keep it.

Reply:
I added it back for its corresponding page (namely `/my/muskathlon`).